cmd/telemetrygen/go.mod

```diff
@@ -1,6 +1,8 @@
 module github.com/open-telemetry/opentelemetry-collector-contrib/cmd/telemetrygen
 
-go 1.22.0
+go 1.22.7
+
+toolchain go1.22.8
 
```

Comment:
@codeboten I think you know something about this: is this fixed on a later version of gRPC?

Reply:
yup, we'll wait until the next release of this lib